library/psa_crypto.c

```diff
@@ -5194,6 +5194,12 @@ psa_status_t psa_aead_update_ad(psa_aead_operation_t *operation,
         goto exit;
     }
 
+    /* No length to add, nothing to do. */
```

Comment:
Nitpick: it's not “no length”, it's a zero length, or no input to add.

Reply:
Yep, that is better.